### PR TITLE
fix: broken contents of "Learn" section

### PR DESCRIPTION
**Pull request by caramis**

### CODE REVIEW

**Review by psy2848048 — COMMENTED** [review]
content/docs/learn/about-xpla.md , line 11

`[staking](glossary.md#staking)`, `[blockchain](glossary.md#blockchain)` are a broken link
Seems to be changed as ref link

**Comment by psy2848048** [comment]
`/docs/learn/about-xpla/#validators`
`blockchain`, `Validator FAQ` links are broken

**Comment by psy2848048** [comment]
`/docs/learn/about-xpla/#staking` 
`Station staking guide`: broken link

**Comment by psy2848048** [comment]
`/docs/learn/about-xpla/#staking`
- Broken MD: `:::{admonition} `
- Broken link `Validator FAQ`

**Comment by psy2848048** [comment]
`/docs/learn/about-xpla/#bonding-staking-and-delegating`
Broken link: `active set`

**Comment by psy2848048** [comment]
`/docs/learn/about-xpla/#unbonding`
Broken MD `:::{Caution}`

**Comment by psy2848048** [comment]
`/docs/learn/about-xpla/#redelegation`
Broken MD `:::{Caution}`

**Comment by psy2848048** [comment]
`/docs/learn/about-xpla/#rewards`
Broken links `Gas`, `Inflation rewards`, & `fee page`

**Comment by psy2848048** [comment]
`/docs/learn/about-xpla/#slashing`
- Broken MD `:::{admonition}`
- Broken link `slashing module`

**Comment by psy2848048** [comment]
`/docs/learn/about-xpla/#governance`
broken link `blockchain` & `Station governance guide`

**Comment by psy2848048** [comment]
`/docs/learn/about-xpla/#voting-process`
Broken link `governance module`

**Comment by psy2848048** [comment]
`/docs/learn/brand-assets/`
Broken links

File: content/docs/learn/station/download/station-extension.md [hunk 1]
```diff
@@ -34,57 +35,51 @@ Use this tutorial to create a new wallet. To connect to an existing wallet, visi
...
 1. Connect to your wallet and click **Buy** next to any token. Purchase your tokens using any exchange that supports Xpla.
 
-   ```{image} /img/screens/ext-wallet.png
-   :class: sd-p-3
-   :width: 500px
+   ```
+   TODO: image
    ```
 
 2. Purchase tokens using your preffered exchange.
```
Comment (jhlee-young) [inline content/docs/learn/station/download/station-extension.md]: typo `preffered` -> `preferred`

File: content/docs/learn/station/_index.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 bookCollapseSection: true
 bookFlatSection: false
-weight: 50
+weight: 40
 title: Station
 ---
```
Comment (jhlee-young) [inline content/docs/learn/station/_index.md:7]: plz check `download tutorial` link, unreachable

**Comment by caramis** [comment]
What a great review. I will quickly reflect the review and push it again.

**Review by psy2848048 — APPROVED** [review]

**Review by jhlee-young — APPROVED** [review]
LGTM

**Pull request merged**